docs/modules/operation/pages/events/mib.adoc

```diff
@@ -0,0 +1,40 @@
+
+= SNMP MIB Compiler
+
+Management Information Base (MIB) files are text files written in Abstact Syntax Notation.
```

Comment:
Should we link to documentation on Abstract Syntax Notation, or would our users generally know what this is?

Reply:
It couldn't hurt. I'll add it in.